src/ResourceManagement/MachineLearning/Microsoft.Azure.Management.MachineLearning/Management/WebServiceManagementClient.cs

```diff
@@ -0,0 +1,227 @@
+        /// Creates a web service object from an AutoRest generated WebService model. Also takes the name of the resource group.
+        /// </summary>
+        /// <param name="webService">The other web service from which we are creating a web service object. Serves as the object Definition internal state object.</param>
+        /// <param name="resourceGroup">The resource group in which the web service is deployed.</param>
+        /// <returns></returns>
+        public WebService CreateWebServiceObject(
+            Microsoft.Azure.Management.MachineLearning.WebServices.Models.WebService webService,
+            string resourceGroup)
+        {
+            var nWebService = new WebService(webService, resourceGroup,this._client);
+            return nWebService;
+        }
+
+        /// <summary>
+        /// Deploys the given web service in the resource group it specifies.
+        /// </summary>
+        /// <param name="webService">The web service to be deployed.</param>
+        /// <param name="force">Optional: If a web service with the same name exists, just go ahead and override it if true. Otherwise, throw an exception.</param>
+        public void DeployWebService(WebService webService, bool force = false)
+        {
+            if (!force)
+            {
+                try
+                {
+                    // If this throws an error, then it doesn't exist.
+                    var existingWebService = this._client.WebServices.Get(webService.ResourceGroupName, webService.Title);
+
+                    throw new InvalidOperationException("force flag was set to false; a web service in the resource group " + 
+                        webService.ResourceGroupName + " with the name " + webService.Title + " already exists.");
+                }
+
+                catch (Exception)
+                {
+                    this._client.WebServices.CreateOrUpdate(webService.Definition, webService.ResourceGroupName,
+                        webService.Title);
+
+                    webService.Keys = this._client.WebServices.ListKeys(webService.ResourceGroupName, webService.Title);
+                }
+            }
+
+            else
+            {
+                this._client.WebServices.CreateOrUpdate(webService.Definition, webService.ResourceGroupName,
+                    webService.Title);
+
+                webService.Keys = this._client.WebServices.ListKeys(webService.ResourceGroupName, webService.Title);
+            }
+        }
+
+        /// <summary>
+        /// Returns a web service from a specified resource group with the specified name.
+        /// </summary>
+        /// <param name="resourceGroupName">The resource group from which to get the web service.</param>
+        /// <param name="webServiceName">The name of the web service to get.</param>
+        /// <returns>The web service.</returns>
+        public WebService GetWebServiceFromResourceGroup(string resourceGroupName, string webServiceName)
```

Comment:
This actually get from resource group by name. What if we just call it GetWebService()?